### PR TITLE
Update Gatling from 3.6.1 to 3.7.2

### PR DESCRIPTION
**Pull request by RomanLebid**
Depends on https://github.com/phiSgr/gatling-ext/pull/2 to be merged first

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version = 1.4.5
\ No newline at end of file
+sbt.version = 1.5.5
```
Comment (RomanLebid) [inline project/build.properties:1]:
```suggestion
sbt.version = 1.5.5

```

**Comment by phiSgr** [comment]
Thank you very much. Sorry I've already worked on that, including ergonomically doing `checkIf`.

The blocking work is the code coverage, if you want I can push my WIP and have you work on that.

**Comment by RomanLebid** [comment]
> Thank you very much. Sorry I've already worked on that, including ergonomically doing `checkIf`.
> 
> The blocking work is the code coverage, if you want I can push my WIP and have you work on that.

Oh, didn't see any PRs or issues open regarding that 😅

Sure, I'd be glad to help. So is there some minimal percentage of code coverage required?

**Comment by phiSgr** [comment]
Well I work on this alone, and the interest in contributing had been low. So I didn't make work in progress visible.

I want every line to be covered in a Gatling test. You can run and check the coverage report in the current master branch. The sbt command is in README.

**Comment by vsandulescu-zd** [comment]
Snyk discovered a high vulnerability for `io.netty:netty-codec`  introduced by `gatling-core` 3.6.1. We are using gatling-core and gatling-grpc for our tests, do you have any idea when this PR could be merged so that we can remediate the vulnerability?

More details from Snyk:
**Detailed paths and remediation**
Introduced through: unknown:unknown@1.0.0-SNAPSHOT › io.gatling:gatling-core@3.6.1 › io.netty:netty-handler@4.1.65.Final › io.netty:netty-codec@4.1.65.Final
Fix: Upgrade to io.gatling:gatling-core@3.7.0 
**Overview**
io.netty:netty-codec is an event-driven asynchronous network application framework.

Affected versions of this package are vulnerable to Denial of Service (DoS). Bzip2Decoder doesn't allow setting size restrictions for decompressed data. An attacker can exploit this by providing a compressed file that, when decompressed, triggers an out-of-memory exception.

**Comment by phiSgr** [comment]
I'll create the new version some time mid Feb.

**Comment by phiSgr** [comment]
BTW, does explicitly specifying a higher netty version work for you? @vsandulescu-zd

**Comment by vsandulescu-zd** [comment]
Thank you for your suggestion @phiSgr. I used a `dependencyOverride` for `netty-codec` and locally I can see the higher version is picked up. Snyk didn't recognize this properly, but we're working on that.

**Comment by jpaulorio** [comment]
I'm using Gatling 3.7.5 and getting the following error. Wondering if this PR would fix it. If I use Gatling 3.5.0 everything works just fine.

`Symbol 'type io.gatling.core.check.FindCheckBuilder' is missing from the classpath.
[error] This symbol is required by 'value com.github.phisgr.gatling.grpc.request.CallDefinition.ts'.
[error] Make sure that type FindCheckBuilder is in your classpath and check for conflicting dependencies with `-Ylog-classpath`.
[error] A full rebuild may help if 'CallDefinition.class' was compiled against an incompatible version of io.gatling.core.check.
`

**Comment by noisypants** [comment]
Hey all.  Interested in gatling 3.7 support as well.  Curious if there's any update on this front?
thanks for the hard work.

**Comment by kvma** [comment]
> Thank you very much. Sorry I've already worked on that, including ergonomically doing `checkIf`.
> 
> The blocking work is the code coverage, if you want I can push my WIP and have you work on that.

Hi @phiSgr, I'd love to help contribute to this and take on the work to improve the code coverage as well.  Did you ever end up pushing this to a WIP branch?

**Comment by phiSgr** [comment]
Closed by 377fadc

Sorry for the delay.

**Pull request closed without merging**